radioactive-decay/content.md

```diff
@@ -0,0 +1,43 @@
+# <div style ="text-align: right">Radioactive Decay</div>
```

Comment:
I think it's great practice for both making PRs but also for practicing reviewing.  That's one aspect of this that will be a little different for all of us, I think, despite our experience with code reviews